runtime/autoload/health/provider.vim

```diff
@@ -437,54 +328,17 @@ function! s:check_python(version) abort
       call health#report_info(printf('pynvim version: %s', current))
     endif
 
-    if s:is_bad_response(current)
-      call health#report_error(
-        \ "pynvim is not installed.\nError: ".current,
-        \ ['Run in shell: '. pip .' install pynvim'])
-    endif
-
```

Comment:
This can never happen: in this branch, we're using a Python interpreter which has already proven it can `import neovim`.